src/lib/ai/mental-llama/MentalLLaMAAdapter.ts

```
@@ -179,290 +241,241 @@
+                if (routingDecision.targetAnalyzer === 'crisis') {
+                  logger.warn(
+                    'CRISIS DETECTED BY TASK ROUTER. Initiating crisis protocol.',
+                    { textSample: text.substring(0, 100) }
+                  )
+                  effectiveCategories = ['depression', 'anxiety', 'stress'] // Broad categories for crisis context
+                  analysisMentalHealthCategory = 'crisis'
+
+                  // Implement proper crisis protocol using our dedicated crisis system
+                  try {
+                    // Import the crisis protocol system
+                    const { CrisisProtocol, CrisisRiskDetector, initializeCrisisProtocol } = require('../crisis')
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)